TEST.md

```diff
@@ -0,0 +1,51 @@
+# How to Setup and Run Tests
+
+MyUsa uses several different tools for testing various aspects of its operation:
+
+* [Rspec](http://rspec.info/), a behavior-driven development testing framework for Ruby
+* [Capybara](http://jnicklas.github.io/capybara/), a library for integration testing
+* [Code Climate](https://codeclimate.com/), a service for automated code reviews
+
+These testing mechanisms uses some additional tools to get what they need done
+
+* [Site Prism](https://github.com/natritmeyer/site_prism), a simple DSL for the DOM within Capybara
```

Comment:
More than that - it's an implementation of the Page Object Model (see http://www.guru99.com/page-object-model-pom-page-factory-in-selenium-ultimate-guide.html ), which is fundamental to how we write our feature tests.